src/openproduct/producten/serializers/product.py

```diff
@@ -36,6 +36,7 @@
             value={
                 "uuid": "da0df49a-cd71-4e24-9bae-5be8b01f2c36",
                 "url": "https://gemeente.open-product.nl/producten/api/v0/producten/da0df49a-cd71-4e24-9bae-5be8b01f2c36",
+                "naam": "verhuurvergunning",
```

Comment:
```suggestion
                "naam": "verhuurvergunning: straatweg 14",
```
verhuurvergunning is the name of the producttype, the product name should be an identifier of the specific product.